assets/js/lib/locations/covidBedLocation.js

```diff
@@ -5,7 +5,7 @@ class CovidBedLocation {
     this.latitude  = parseFloat(dataRow[2]);
     this.longitude = parseFloat(dataRow[3]);
     this.url       = dataRow[4];
-    if ((isNaN(this.latitude) || isNaN(this.longitude)) && Boolean(this.url)) {
+    if ((isNaN(this.latitude) || isNaN(this.longitude)) && Boolean(this.url) && this.url.includes("@")) {
```

Comment:
It might be the case that a false URL also has an `@` in it. It's best if we play more safe with this parsing. How about something like this:
```js
if ((isNaN(this.latitude) || isNaN(this.longitude)) && Boolean(this.url)) {
  try { 
     var url_location = this.url.split('@')[1].split('z')[0].split(',');
     this.latitude = parseFloat(url_location[0]);
     this.longitude = parseFloat(url_location[1]);
  } catch (e) { 
     console.log(`Error in parsing URL for ${this.type} for title: ${this.title}`);
     console.log(e);
  }
}